IntelliTect.TestTools.WindowsTestWrapper/ExampleProjects/Notepad.AutomatedTests/Helpers/NotepadWindowHelpers.cs

```diff
@@ -10,6 +12,37 @@ public class NotepadWindowHelpers
         private readonly NotepadWindow _NotepadWindow = new NotepadWindow();
         private readonly SaveAsWindow _SaveAsWindow = new SaveAsWindow();
 
+        public bool ValidateTextFile(string location, string fileName, string expectedText)
+        {
+            bool comparisonPass = false;
+            string combinedPath = Path.Combine( location, fileName );
+            if ( combinedPath == expectedText )
+            {
+                comparisonPass = true;
+            }
+            return comparisonPass;
+        }
```

Comment:
It would have helped to keep in the File.ReadAllText when I moved this over and refactored it a bit... That might have cleared up some confusion *facepalm*